figures/tasks.py

```diff
@@ -378,8 +376,8 @@ def populate_monthly_metrics_for_site(site_id):
 @shared_task
 def run_figures_monthly_metrics():
     """
-    TODO: only run for active sites. Requires knowing which sites we can skip
+    Populate monthly metrics for all sites.
```

Comment:
Technically, this is is only *all* sites when a filtered sites backend is not used